### ISSUE TITLE
Bug list is missing items

### ISSUE THREAD

**Issue by genodeftest** [abrt/faf#582]
TL;DR: https://retrace.fedoraproject.org/faf/problems/?component_names=gnome-shell&daterange=2017-07-05%3A2017-07-22 is missing https://retrace.fedoraproject.org/faf/reports/1777672/

I'm using the FAF instance at https://retrace.fedoraproject.org/faf/

Steps to reproduce:
1. open https://retrace.fedoraproject.org/faf/problems/
2. insert a component, e.g. gnome-shell
3. press "Query"

What happens:
FAF lists some items, but not all of them, e.g. https://retrace.fedoraproject.org/faf/reports/1777672/ is missing

What should happen:
FAF should list all items matching the query

This issue may be related to #236.

**Comment by mgrabovsky** [comment]
This appears to stem from a confusion between _reports_ and _problems_. The listing of problem (first link) cannot point to a report (second link).
